src/borg/testsuite/compress.py

```diff
@@ -116,40 +116,53 @@ def test_auto():
 
 
 @pytest.mark.parametrize(
-    "specs, c_type, result_range, obfuscate_factor",
+    "specs, c_type, result_range, obfuscation_factor",
     [
         ("obfuscate,1,none", CNONE, 50, 101),
-        ("obfuscate,110,none", CNONE, None, 1024),
         ("obfuscate,2,lz4", LZ4, 10, 1001),
         ("obfuscate,6,zstd,3", ZSTD, 90, 10000001),
         ("obfuscate,2,auto,zstd,10", Auto, 10, 1001),
```

Comment:
to make the relation more obvious, you could use `10**1`, `10**2`, `10**6`, `10**2` here and use `10*x+1` below.